src/app/chip_data_model.gni

```diff
@@ -18,14 +18,24 @@ import("${chip_root}/src/lib/core/core.gni")
 
 _app_root = get_path_info(".", "abspath")
 
+_zap_cluster_list_script = get_path_info("zap_cluster_list.py", "abspath")
+
 # Defines a source_set for CHIP data model.
 #
 # Arguments:
 #  zap_pregenerated_dir
 #    Path to the ZAP "gen" dir.
 #
+#  zap_file
+#    Path to the ZAP input file.
+#
 #  cluster_sources
-#    Names of the clusters to compile.
+#    Names of the clusters directories to compile.
+#    Deprecated, specify zap_file instead.
+#
+#  remove_cluster_sources
```

Comment:
This seems to be unused in practice.  Is it no longer needed now that we are checking the "enabled" flag?

Reply:
No longer needed at the moment. But maybe someone might find a new use for it? Happy to remove.

Reply:
I'm fine either way; just odd to have a comment saying "remove when it's no longer needed" on something that is in fact not currently needed.  ;)

Reply:
Good point.